app/models/manageiq/providers/ibm_cloud/object_storage/storage_manager.rb

```diff
@@ -0,0 +1,211 @@
+class ManageIQ::Providers::IbmCloud::ObjectStorage::StorageManager < ManageIQ::Providers::StorageManager
+  require_nested :Refresher
+  require_nested :RefreshWorker
+
+  supports :ems_storage_new
+  include ManageIQ::Providers::StorageManager::ObjectMixin
+
+  def self.params_for_create
+    @params_for_create ||= {
+      :fields => [
+        {
+          :component => 'sub-form',
+          :id        => 'endpoints-subform',
+          :name      => 'endpoints-subform',
+          :title     => _("Endpoints"),
+          :fields    => [
+            :component => 'tabs',
+            :name      => 'tabs',
+            :fields    => [
+              {
+                :component => 'tab-item',
+                :id        => 'default-tab',
+                :name      => 'default-tab',
+                :title     => _('Default'),
+                :fields    => [
+                  {
+                    :component              => 'validate-provider-credentials',
+                    :id                     => 'endpoints.default.valid',
+                    :name                   => 'endpoints.default.valid',
+                    :skipSubmit             => true,
+                    :isRequired             => true,
+                    :validationDependencies => %w[type zone_id uid_ems],
+                    :fields                 => [
+                      {
+                        :component  => "text-field",
+                        :name       => "provider_region",
+                        :id         => "provider_region",
+                        :label      => _("Region"),
+                        :isRequired => true,
+                        :validate   => [{:type => "required"}],
+                      },
+                      {
+                        :component  => "text-field",
+                        :name       => "uid_ems",
+                        :id         => "uid_ems",
+                        :label      => _("Resource Instance Id"),
+                        :isRequired => true,
+                        :validate   => [{:type => "required"}],
+                      },
+                      {
+                        :component  => "text-field",
+                        :name       => "endpoints.default.url",
+                        :id         => "endpoints.default.url",
```

Comment:
Is a URL required or can it be built from the region?  Assuming we are talking to the service API here